Apps/WebClient/src/ClientApp/src/models/hospitalVisitTimelineEntry.ts

```diff
@@ -34,9 +34,12 @@ export default class HospitalVisitTimelineEntry extends TimelineEntry {
         this.admitDateTime = new DateWrapper(model.admitDateTime, {
             hasTime: true,
         });
-        this.endDateTime = new DateWrapper(model.endDateTime, {
-            hasTime: true,
-        });
+        this.endDateTime =
```

Comment:
Wouldn't this change be better done on the C# side?

Reply:
The null endDateTime from the back end is a legitimate state so it had to be handled on the front end.